### PR TITLE
Removed storing password

### PR DESCRIPTION
**Pull request by kmoll**

### CODE REVIEW

File: github_api.module [hunk 1]
```diff
@@ -37,13 +38,17 @@ function github_api_client($auth = TRUE) {
   $client = new Client($cache_obj);
 
   if ($auth) {
-    $token = $config->get('github_api_token');
+    // $token = $config->get('github_api_token');.
```
Comment (skwashd) [inline github_api.module]: If this isn't needed it should be removed.

**Review by skwashd — CHANGES_REQUESTED** [review]
@kmoll overall this is looking good. Please remove the commented line and squash this into a single commit and then I think it is ready go.

**Comment by kmoll** [comment]
@skwashd I updated this, we needed the line, so I kept it and removed the comments.

**Review by skwashd — APPROVED** [review]
Looks good to me. Thanks for submitted the patch @kmoll.

**Pull request merged**